md/release-notes.md

```diff
@@ -144,3 +146,7 @@ To avoid memory issues with large bar files deployment the default xmx value has
 ### Complex data-types generation have been removed
 
 This feature was used to generate Java POJOs and XSD in Subscription editions. It is recommend to add your own Java model as jar file in the project classpath or create your [data model using Groovy objects](groovy-in-bonita.md#create-data-model).
+
+### Description field on widgets
+
+Add documentation to your pages explaining how and what your widgets do thanks to the new optional field "description" and use this information to [generate documentation](release-notes.md#project-documentation).
```

Comment:
What about:
Add documentation to your pages by providing information on a widget (e.g. how to use it), thanks to the new Description field. Use this information to [generate documentation](release-notes.md#project-documentation).